decidim-comments/app/models/decidim/comments/seed.rb

```diff
@@ -0,0 +1,14 @@
+module Decidim
```

Comment:
That happens when the comment is added on the first *missing* line of a PR. Stupid hound doesn't know if we fixed it or not.